### PR TITLE
Update external-oasis-repos.md

### PR DESCRIPTION
**Pull request by steige2**
Added security related changes regarding repository blanking.

### CODE REVIEW

File: docs/policy/external-oasis-repos.md [hunk 1]
```diff
@@ -31,8 +31,11 @@ OSG Responsibilities
 * OSG will provide a best-effort mirror of the full contents of the external repo.  We will attempt to provide best-effort integrity of the
   object contents, but assume users of the Stratum-1 will do further integrity checking.  No SLA is provided covering potential data corruptions.
 * OSG will provide best-effort notification to the mirrored repository in case OSG detects a service outage of the external repo.
-* In the event of a reported security incident, OSG will replace the repository contents with an empty directory, signed by an OSG-managed key.
-  This will be done under the direction of the security team.
+* In the event of a security incident, the operations group at Indiana University
```
Comment (brianhlin) [inline docs/policy/external-oasis-repos.md]: Why limit this to Indiana?

**Comment by steige2** [comment]
I’m assigning a responsibility to a specific group and documenting approved policy.
On this one in particular we can’t have any room for wondering who does what.
S

Scott Teige PhD
Grid Operations Center Manager
Indiana University Research Technologies
steige@iu.edu<mailto:steige@iu.edu>
Office: +1 812 856 7331




On Nov 8, 2017, at 11:07 AM, Brian Lin <notifications@github.com<mailto:notifications@github.com>> wrote:


@brianhlin commented on this pull request.

________________________________

In docs/policy/external-oasis-repos.md<https://github.com/opensciencegrid/technology/pull/361#discussion_r149714624>:

> @@ -31,8 +31,11 @@ OSG Responsibilities
 * OSG will provide a best-effort mirror of the full contents of the external repo.  We will attempt to provide best-effort integrity of the
   object contents, but assume users of the Stratum-1 will do further integrity checking.  No SLA is provided covering potential data corruptions.
 * OSG will provide best-effort notification to the mirrored repository in case OSG detects a service outage of the external repo.
-* In the event of a reported security incident, OSG will replace the repository contents with an empty directory, signed by an OSG-managed key.
-  This will be done under the direction of the security team.
+* In the event of a security incident, the operations group at Indiana University


Why limit this to Indiana?

**Comment by brianhlin** [comment]
In that case, "Operations" should be specific enough. What if a non-IU ops member takes on external OASIS repo responsibilities? What if all IU ops members are unavailable? Narrowing the scope to that extent seems unnecessary here.

**Comment by steige2** [comment]
What if a non-IU ops member takes on external OASIS repo responsibilities?

In the unlikely event of a transfer of service, a transfer of responsibilities would be included. The receiving site would be responsible for updating the documentation.


What if all IU ops members are unavailable?

We are available 24x7x365 if needed, 317-278-9699 will connect you to the GRNOC. They have a call list to go through until they reach someone. This mechanism has been used (once or twice per year) and has always worked. (I am at the top of the list…) I believe we are the only subgroup of ops with this property but I may be mistaken.


Scott Teige PhD
Grid Operations Center Manager
Indiana University Research Technologies
steige@iu.edu<mailto:steige@iu.edu>
Office: +1 812 856 7331




On Nov 8, 2017, at 12:39 PM, Brian Lin <notifications@github.com<mailto:notifications@github.com>> wrote:


In that case, "Operations" should be specific enough. What if a non-IU ops member takes on external OASIS repo responsibilities? What if all IU ops members are unavailable? Narrowing the scope to that extent seems unnecessary here.

File: docs/policy/external-oasis-repos.md (hunk 1 above)
Comment (osg-cat) [reply docs/policy/external-oasis-repos.md]: Agreed. Based on our discussion about Operations teams at the November 1st Area Coordinators meeting, it seems like this point should be defined in terms of a role, a responsibility, or a capability. Think SciTokens!

**Comment by brianhlin** [comment]
This has been languishing for weeks now so I just made the change myself.

**Pull request merged**